webpay/controllers/front/validate.php

```diff
@@ -163,7 +163,7 @@ private function processPayment($webpayTransaction, $cart)
             $currency = Context::getContext()->currency;
             $OKStatus = Configuration::get('WEBPAY_DEFAULT_ORDER_STATE_ID_AFTER_PAYMENT');
             if ($OKStatus === "0") {
-                $OKStatus = Configuration::get('PS_OS_PREPARATION');
+                $OKStatus = Configuration::get('PS_OS_PAYMENT');
```

Comment:
```suggestion
                $OKStatus = Configuration::get('PS_OS_PREPARATION');
```
This change could make the update not backwards compatible.